pkg/utils/utils.go

```diff
@@ -193,9 +195,34 @@ func GetS3PETMFromParamsMap(params map[string]interface{}, logger logrus.FieldLo
 		return nil, errors.New("Missing bucket param, cannot initialize S3 PETM")
 	}
 
-	sess := session.Must(session.NewSession(&aws.Config{
-		Region: aws.String(region),
-	}))
+	useMinio := true
+	s3Url, ok := params["s3Url"].(string)
```

Comment:
Let's make sure `s3Url` and `pathSytle` are either set or unset for both together. If set, use minio. If unset, use regular aws s3. Otherwise, fail with error, like, "Unknown S3 repository is specified"

Reply:
It's OK to have s3URL set and pathStyle not set - in that case we should use the s3URL as "endpoint" and se pathStyle to false.  S3 has two ways to specify a bucket - originally, the bucket name was part of the path but they have moved towards making it part of the DNS name.  The bucket in the name works well for AWS as it lets them do their scaling via DNS rather than having to read the path and then forward, but it is hard to implement at smaller scales (e.g. Minio) since we would need a special DNS setting to send all those different host names to the minio server.  

Here's two example for bucket jsb-public
This is the path style: https://s3.amazonaws.com/jsb-public/classic_amazon_door_desk.png
This is the DNS name style: https://jsb-public.s3.amazonaws.com/classic_amazon_door_desk.png

It's possible to use either style regardless of whether the S3 endpoint URL (e.g. s3.amazonaws.com or csm-minio.eng.vmware.com) is set, so I don't want to link them together.